docs/arcadia-users-group/20230228-intro-to-python-1/problem_set_1.ipynb

```diff
@@ -0,0 +1,588 @@
+{
```

Comment:
<p>What is the goal here? The ability for someone else to read a piece of code? Or do you actually care about the "efficiency" of it? Or do you think this code is not readable enough and making a couple changes would make it more readable?</p><p> </p><p>Depending on the goal, this may be not worthwhile for a beginner. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Arcadia-Science/arcadia-computational-training/pull/50/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
I'm trying to accomplish a couple of things, perhaps not too effectively here: provide practice with reading and understanding code they didn't write, understanding enough to refactor it, and challenging them to write the code in both a more efficient and more readable way. I'll omit this problem for now, as perhaps those things are easier done with practice.